docs/docs/pods.md

```diff
@@ -0,0 +1,572 @@
+      "healthCheck": null,
+      "volumeMounts": [],
+      "artifacts": [],
+      "labels": null,
+      "lifecycle": null
+    }
+  ],
+  "secrets": null,
+  "volumes": [],
+  "networks": [
+    {
+      "mode": "host"
+    }
+  ],
+  "scaling": {
+    "kind": "fixed",
+    "instances": 10,
+    "maxInstances": null
+  },
+  "scheduling": {
+    "backoff": {
+      "backoff": 1,
+      "backoffFactor": 1.15,
+      "maxLaunchDelay": 3600
+    },
+    "upgrade": {
+      "minimumHealthCapacity": 1,
+      "maximumOverCapacity": 1
+    },
+    "placement": {
+      "constraints": [],
+      "acceptedResourceRoles": []
+    }
+  }
+}
+```
+
+## A Pod that Uses Ephemeral Volumes
+
+The following pod definition specifies an ephemeral volume called `v1`.
+
+```json
+{
+  "id": "/with-ephemeral-vol",
+  "scaling": { "kind": "fixed", "instances": 1 },
+  "containers": [
+    {
+      "name": "ct1",
+      "resources": {
+        "cpus": 0.1,
+        "mem": 32
+      },
+      "exec": { "command": { "shell": "while true; do echo the current time is $(date) > ./jdef-v1/clock; sleep 1; done" } },
+      "volumeMounts": [
+        {
+          "name": "v1",
+          "mountPath": "jdef-v1"
+        }
+      ]
+    },
+    {
+      "name": "ct2",
+      "resources": {
+        "cpus": 0.1,
+        "mem": 32
+      },
+      "exec": { "command": { "shell": "while true; do cat ./etc/clock; sleep 1; done" } },
+      "volumeMounts": [
+        {
+          "name": "v1",
+          "mountPath": "etc"
+        }
+      ]
+    }
+  ],
+  "networks": [
+    { "mode": "host" }
+  ],
+  "volumes": [
+    { "name": "v1" }
+  ]
+}
+```
+
+## IP-per-Pod Networking
+
+The following pod definition specifies a virtual (user) network named `my-virtual-network-name`.
+
+```json
+{
+  "id": "/pod-with-virtual-network",
+  "scaling": { "kind": "fixed", "instances": 1 },
+  "containers": [
+    {
+      "name": "sleep1",
+      "exec": { "command": { "shell": "sleep 1000" } },
+      "resources": { "cpus": 0.1, "mem": 32 }
+    }
+  ],
+  "networks": [ { "mode": "container", "name": "my-virtual-network-name" } ]
+}
+```
+
+This pod declares a “web” endpoint that listens on port 80.
+
+```json
+{
+  "id": "/pod-with-endpoint",
+  "scaling": { "kind": "fixed", "instances": 1 },
+  "containers": [
+    {
+      "name": "sleep1",
+      "exec": { "command": { "shell": "sleep 1000" } },
+      "resources": { "cpus": 0.1, "mem": 32 },
+      "endpoints": [ { "name": "web", "containerPort": 80, "protocol": [ "http" ] } ]
+    }
+  ],
+  "networks": [ { "mode": "container", "name": "my-virtual-network-name" } ]
+}
+```
+
+This pod adds a health check that references the “web” endpoint; mesos will execute an HTTP request against `http://<master-ip>:80/ping`.
+
+```json
+{
+  "id": "/pod-with-healthcheck",
+  "scaling": { "kind": "fixed", "instances": 1 },
+  "containers": [
+    {
+      "name": "sleep1",
+      "exec": { "command": { "shell": "sleep 1000" } },
```

Comment:
if anyone actually runs this it will fail health checks because "sleep" isn't a web server. consider updating this example to use nginx, or some other "real" web service for which there is a widely available image